crates/reporting/tests/test_reporting.rs

```diff
@@ -8547,6 +8546,18 @@ All branches in an `if` must have the same type!
             "#
         ),
         @r###"
+    ── OVERLOADED SPECIALIZATION ───────────────────────────── /code/proj/Main.roc ─
+
+    This ability member specialization is already claimed to specialize
+    another opaque type:
+
+    7│  Two := {} has [Hash {hash}]
+                             ^^^^
+
+    Previously, we found it to specialize `hash` for `One`.
+
+    Ability specializations can only provide implementations for one
+    opauqe type, since all opaque types are different!
```

Comment:
typo of `opaque`

Reply:
Nice catch. Weird that spellcheck didn't see that.